services/director-v2/tests/unit/conftest.py

```diff
@@ -92,32 +96,47 @@ def dask_local_cluster(monkeypatch: MonkeyPatch) -> LocalCluster:
 
 
 @pytest.fixture
-def dask_spec_local_cluster(monkeypatch: MonkeyPatch) -> SpecCluster:
+def cluster_id() -> NonNegativeInt:
+    return random.randint(0, 10)
+
+
+@pytest.fixture
+def cluster_id_resource(cluster_id: NonNegativeInt) -> str:
+    return f"CLUSTER_{cluster_id}"
+
+
+@pytest.fixture
+def dask_spec_local_cluster(
+    monkeypatch: MonkeyPatch, cluster_id_resource: str
+) -> SpecCluster:
     # in this mode we can precisely create a specific cluster
     workers = {
         "cpu-worker": {
             "cls": Worker,
-            "options": {"nthreads": 2, "resources": {"CPU": 2, "RAM": 48e9}},
+            "options": {
+                "nthreads": 2,
+                "resources": {"CPU": 2, "RAM": 48e9, cluster_id_resource: 1},
```

Comment:
Q: what are the units for CPU and RAM ? uCPUs? and Bytes? Some of your mockups are e.g. 1e-9 ... and I do not understand them

Reply:
1CPU = one cpu. but you could use fractions, else they are called nano_cpus I think.
RAM is in byte